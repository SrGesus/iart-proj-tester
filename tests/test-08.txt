VE	FD	VE	LH	FB
LV	FC	BB	LH	VE
BE	VB	BC	FC	LV
VE	BD	BD	VD	LV
FC	VE	FE	VE	FD