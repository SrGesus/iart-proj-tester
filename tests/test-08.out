VB	FE	VB	LH	FE
LV	FB	BD	LH	VE
BD	VC	BD	FE	LV
VD	BB	BC	VE	LV
FD	VC	FD	VC	FC
